indexer/eventprocessor.go

```diff
@@ -47,25 +47,34 @@ func (e *EventProcessor) Run() {
 	for ; true; <-ticker.C {
 		var err error
 		currentSlot, currentHash, err = currentChainStatus(e.db)
-		if err == nil {
+		if err != nil && err != gorm.ErrRecordNotFound {
+			log.Error("fail query chain status", "error", err)
+			continue
+		} else if err == nil {
 			break
-		} else if err == gorm.ErrRecordNotFound {
+		}
+
+		if err := e.db.Transaction(func(dbTx *gorm.DB) error {
+			if err := e.processGenesisValidators(dbTx); err != nil {
+				return err
+			}
+
 			block, err := e.node.BlockBySlot(e.ctx, 0)
```

Comment:
Instead of process slot 0 here, we could just set currentHash to zero and let processEvents() handle it like normal case?